lib/SIL/SILPrinter.cpp

```diff
@@ -243,10 +243,13 @@ static void printValueDecl(ValueDecl *Decl, raw_ostream &OS) {
   printFullContext(Decl->getDeclContext(), OS);
   assert(Decl->hasName());
 
-  if (Decl->isOperator())
-    OS << '"' << Decl->getName() << '"';
-  else
-    OS << Decl->getName();
+  if (Decl->isOperator()) {
+    OS << '"' << Decl->getBaseName() << '"';
+  } else if (Decl->getBaseName() == Decl->getASTContext().Id_subscript) {
+    OS << '`' << Decl->getBaseName() << '`';
```

Comment:
Also, it seems like for SIL it might make sense to have one consistent escaping syntax. Quotes might be fine. @swiftix, @gottesmm?